src/madhuraMJun20/SumOfNumbersASCII.java

```diff
@@ -0,0 +1,29 @@
+/*WAP to sum all the numbers in given string, without using Character class method. 
+(Note : Use Ascii value concept) .
+Input: J7u9y8h1h8j4b7j3j6
+Ouput: Sum of all numbers in String is - 54.
+A-Z = 65-90
+a-z = 97-122
+0-9 = 48-57
+*/
+package madhuraMJun20;
+
+public class SumOfNumbersASCII {
+	void sumOfNumbersUsingAsciiValue(String input) {
+		int sum = 0;
+		// int temp1 ='0';
+		// int temp2='9';
+		for (int index = 0; index < input.length(); index++) {
+			if (input.charAt(index) >= '0' && input.charAt(index) <= '9') {
+				// char ch = input.charAt(index);
+				sum += Integer.parseInt(String.valueOf(input.charAt(index)));
```

Comment:
sum += (str.charAt(index) - '0'); You can use this as well. 
Or  sum = sum + Character.getNumericValue(str.charAt(index);

Reply:
Ok mam. I will remember this next time. Thank you!